### PR TITLE
Fix relay/contracts changes

### PR DESCRIPTION
**Pull request by skudasov**
Validator/Proxies and chaos tests, do not merge.

### CODE REVIEW

File: tests/e2e/env.go [hunk 1]
```diff
@@ -19,16 +30,20 @@ func NewChainlinkTerraEnv() *environment.Config {
...
 						},
 					},
+					"db": db,
 					"env": map[string]interface{}{
+						"EVM_ENABLED":                 "false",
+						"EVM_RPC_ENABLED":             "false",
+						"TERRA_ENABLED":               "true",
 						"eth_disabled":                "true",
-						"CHAINLINK_DEV":               "true",
+						"CHAINLINK_DEV":               "false",
```
Comment (krebernisak) [inline tests/e2e/env.go:46]: This seems useful in e2e tests, why remove it?

File: tests/e2e/env.go [hunk 2]
```diff
@@ -3,7 +3,18 @@ package e2e
...
+	var db map[string]interface{}
+	if stateful {
+		db = map[string]interface{}{
+			"stateful": true,
+			"capacity": "2Gi",
+		}
+	} else {
+		db = map[string]interface{}{
+			"stateful": false,
+		}
```
Comment (krebernisak) [inline tests/e2e/env.go:16]: To shed some code this could be the default case, and if stateful we overwrite.

File: tests/e2e/ocr2.go [hunk 3]
```diff
@@ -220,5 +200,5 @@ func (t *OCRv2) TransferOwnership(to string) error {
 }
 
 func (t *OCRv2) Address() string {
-	return t.address.String()
+	return t.Addr.String()
```
Comment (krebernisak) [inline tests/e2e/ocr2.go:203]: Why make `Addr` public?

File: tests/e2e/env.go (hunk 1 above)
Comment (skudasov) [reply tests/e2e/env.go]: If I set it to "true" where we test it when it's maybe set to "false" in prod?

File: tests/e2e/chaos/chainlink-relay-terra.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  mockserver:
+    index: 3
+  chainlink:
+    index: 4
+    values:
+      replicas: 5
+      chainlink:
+        image:
+          image: "public.ecr.aws/z0b1w9r9/chainlink"
+          version: "candidate-develop-terr-qa-test-b64-decode-signers-3.233d4d167de902af193b8c83cc79e707d7e74541"
```
Comment (jmank88) [inline tests/e2e/chaos/chainlink-relay-terra.yaml:16]: `release/develop-terr-qa-test-feb-02`
https://github.com/smartcontractkit/chainlink/compare/release/develop-terr-qa-test-feb-02
```suggestion
          version: "candidate-develop-terr-qa-feb-02"
```

**Pull request merged**